archunit/src/main/java/com/tngtech/archunit/core/importer/DomainBuilders.java

```diff
@@ -219,15 +220,28 @@ SELF withParameters(List<JavaType> parameters) {
             this.parameters = parameters;
             return self();
         }
+        
+        SELF withExceptions(List<JavaType> exceptions) {
```

Comment:
Same here, just because ASM calls it `exceptions` doesn't mean we have to pull this into `JavaCodeUnitBuilder`